src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1116,7 +1116,12 @@ internal SingleLookupResult CheckViability(Symbol symbol, int arity, LookupOptio
                 ? ((AliasSymbol)symbol).GetAliasTarget(basesBeingResolved)
                 : symbol;
 
-            if (WrongArity(symbol, arity, diagnose, options, out diagInfo))
+            // Check for external symbols marked with 'Microsoft.CodeAnalysis.Embedded' attribute
+            if (!this.Compilation.SourceModule.Equals(unwrappedSymbol.ContainingModule) && unwrappedSymbol.IsHiddenByCodeAnalysisEmbeddedAttribute())
```

Comment:
Curious - why "external" part makes a difference? I thought the embedded symbols are not bindable regardless of where defined.

---
In reply to: [113533730](https://github.com/dotnet/roslyn/pull/18715#discussion_r113533730) [](ancestors = 113533730)